lte/gateway/deploy/agw_install_docker.sh

```diff
@@ -118,4 +118,14 @@ else
   su - $MAGMA_USER -c "sudo ansible-playbook -v -e \"MAGMA_ROOT='/opt/magma' OUTPUT_DIR='/tmp'\" -i $DEPLOY_PATH/agw_hosts --tags agwc $DEPLOY_PATH/magma_docker.yml"
 fi
 
+# check if we are on ARM system
+if [ "$(uname -m)" == "aarch64" ]; then
+  sed -i 's/OPTIONAL_ARCH_POSTFIX=/OPTIONAL_ARCH_POSTFIX=_arm/' /var/opt/magma/docker/.env
+fi
+
+if [ "${MAGMA_VERSION}" == "v1.8" ]; then
+  sed -i 's,DOCKER_REGISTRY=,DOCKER_REGISTRY=docker.artifactory.magmacore.org/,' /var/opt/magma/docker/.env
+  sed -i 's/IMAGE_VERSION=latest/IMAGE_VERSION=1.8.0/' /var/opt/magma/docker/.env
+fi
+
```

Comment:
We don't need this now. We have already done change on Ansible code for making these changes.